FileGen4.py

```diff
@@ -0,0 +1,385 @@
+from docx import Document
+from docx.shared import Inches, Pt, RGBColor, Length
+from docx.enum.text import WD_COLOR_INDEX
+from docx.enum.dml import MSO_THEME_COLOR
+from docx.dml.color import ColorFormat
+from docx.text.run import Font, Run
+from docx.enum.text import WD_ALIGN_PARAGRAPH #does not work for ryans pycharm
+from docx.enum.text import WD_PARAGRAPH_ALIGNMENT #this one works for ryans pycharm
+from docx.oxml.shared import OxmlElement, qn
+from copy import deepcopy
+import datetime
+import textwrap
+
+#pip3 install python-docx
+#pip3 install DateTime
+
+# This class takes a template file and then fills in the correct data. It then will
+# get emailed for approval.
+# template file must be in the same directory or its location specified.
+# To instance form gen one should provide the correct OC insturctor name, OC department,
+# military_course, and the oc_course
+
+#creating a FileGen object will fill out the top of the form.
+
+#----- About: Nested_Tables_Form_Template_Version_4_11_2019-----
+#If things dont line up you have to make sure that the margin for the cell where the mlt outcomes go is lined up 
+#so that the courser is just showing. If it still doesnt line up then you will have to mess with it. But I am hoping
+#that the formating follows the form template.
+
+class FileGen:
+
+        # initializing variables to be used throught the generator
+    def __init__(self, oc_course, jst_course, reviewer):
+        # test.docx should be the name of the template file.
+        self.doc = Document('Nested_Tables_Form_Template_Version_4_11_2019')# Have to use this form template.
```

Comment:
Nested_Tables_Form_Template_Version_4_11_2019 needs to be changed to Nested_Tables_Form_Template_Version_4_11_2019.docx